lib/web_ui/lib/src/engine/html/dom_canvas.dart

```diff
@@ -338,9 +339,7 @@ SVGSVGElement pathToSvgElement(
   // In Firefox some SVG typed attributes are returned as null without a
   // setter. So we use strings here.
   final SVGSVGElement root = createSVGSVGElement()
-    ..setAttribute('width', '${width}px')
-    ..setAttribute('height', '${height}px')
-    ..setAttribute('viewBox', '0 0 $width $height');
```

Comment:
I think this means we can remove the `width` and `height` parameters from this function, and transitively all code that computes and passes these values.